test/bdd/pkg/waci/waci_issuance_sdk_steps.go

```diff
@@ -70,6 +81,7 @@ func (i *IssuanceSDKSteps) SetContext(ctx *context.BDDContext) {
 }
 
 // RegisterSteps registers the BDD test steps on the suite.
+// Note that VC proofs are not checked in this test suite.
```

Comment:
Should I also check VC proofs? (if so, I can do that in my next PR) @sudeshrshetty @rolsonquadras

Reply:
that's ok, but nice to have feature. If it can be quick then do it.